haystack/nodes/reader/farm.py

```diff
@@ -108,7 +108,7 @@ def __init__(
                              Can be helpful to disable in production deployments to keep the logs clean.
         :param duplicate_filtering: Answers are filtered based on their position. Both start and end position of the answers are considered.
                                     The higher the value, answers that are more apart are filtered out. 0 corresponds to exact duplicates. -1 turns off duplicate removal.
-        :param use_confidence_scores: Sets the type of score that is returned with every predicted answer.
+        :param use_confidence_scores: Determines the type of score that is used for ranking a predicted answer.
```

Comment:
We could add a hint like "This can affect the ranking of no_answers in particular." to make it a bit easier to grasp what that actually means.